scripts/events_module/relationship/romantic_events.py

```diff
@@ -452,8 +454,11 @@ def handle_breakup(cat_from: Cat, cat_to: Cat) -> bool:
         # Determine if this is a nice breakup or a fight breakup
         # TODO - make this better
         had_fight = not int(random.random() * 3)
+        decided_to_be_friends = not int(random.random() * 3)
+        lost_feelings = not int(random.random() * 2)
+        bad_breakup = not int(random.random() * 5)
+        chill_breakup = not int(random.random() * 5)
```

Comment:
```suggestion
        breakup_type = random.choices(["had_fight", "decided_to_be_friends", "lost_feelings", "bad_breakup", "chill_breakup"], [3, 3, 2, 5, 5])
```
I'd suggest something like this instead